### ISSUE TITLE
[suggestion] better honeycomb brood blocks

### ISSUE THREAD

**Issue by kaboom24** [TelepathicGrunt/Bumblezone#301]
when the larva from a honeycomb brood block becomes a baby bee the brood block should become empty and is refilled when a bee that is ready for a larva it should go to the nearest brood block and transform it into a phase 1 honeycomb brood block also bees outside of the bumblezone should go inside the bee nest/hive when they are ready to lay a brood block then after the same amount of time it takes for a phase one honeycomb brood to make a baby bee a baby bee will spawn outside the hive

**Comment by TelepathicGrunt** [comment]
I do not want to hack a new goal into bee mobs to drive them to my block. Furthermore, bee breeding with flowers is much easier and faster than trying to force bees to brood blocks and then having to grow the brood blocks. It would make the brood blocks literally pointless.

 Also, what would make a bee even ready for a brood block? If they mate with another bee, they spawn a baby bee right there without brood blocks so mating won’t work. If it random over time, then the bee is creating a baby in brood blocks without mating which is weird 

Lastly, the brood blocks were setup the currently way specifically to both be an easy renewable source of bees and also spawn bees much faster if players with wrath of hive are near the brood blocks. If I were to empty out the brood blocks and force require bees to repopulate, I lose this neat wrath behavior as the brood blocks won’t keep spawning bees quickly. 

Also, when outside Bumblezone, the Bumblezone dimension is not loaded. Bees going into hives are not teleported to Bumblezone. If they were, then you effectively just lose your beehive bees for good because they are no longer stored in the block and get trapped in Bumblezone and can’t exit Bumblezone because they would need to go outside would bounds to do so. Also, loading Bumblezone chunks while player isn’t in Bumblezone is terrible and will hurt server performance for no good gain that the player can see 

basically, I already thought about all this long ago and decided on the current implementation for many reason.
